README.md

```diff
@@ -246,8 +269,14 @@ There are two ways to specify the role: use the built-in role or create a custom
     ```bash
     AZURE_CLIENT_ID=`az ad sp list --display-name "velero" --query '[0].appId' -o tsv`
     ```
+3. Assign Additional permissoins to the service principal (For Latest Azure Plugin >1.7.0 with useAAD=true)
 
-3. Now you need to create a file that contains all the relevant environment variables. The command looks like the following:
+    If you chose the AAD route, this is an additional permissions required for the service principal to be able to access the storage account.
+    ```bash
+    az role assignment create --assignee $AZURE_CLIENT_ID --role "Storage Blob Data Contributor" --scope /subscriptions/$AZURE_SUBSCRIPTION_ID
```

Comment:
If users provide `storageAccountURI` in BSL, is the role `Storage Blob Data Contributor` still needed?

Reply:
Yes, that's needed for the actual data plane access.

Reply:
@ywk253100 addressed most of your comments - can you have a look now?

Reply:
oh ohkay
on it

Reply:
Fixed it, and squashed commits.